common_source/modules/mat_elem/eos_param_mod.F90

```diff
@@ -48,11 +48,11 @@ module eos_param_mod
       
       type eos_param_
         character(len=nchartitle) :: title = ''  !< eos model input name
-        integer     :: nuparam                   !< number of real value paraameters
-        integer     :: niparam                   !< number of int value parameters
-        integer     :: nuvar                     !< number of internal state variables
-        integer     :: nfunc                     !< number of local functions in material
-        integer     :: ntable                    !< number of local function tables
+        integer :: nuparam = 0                   !< number of real value paraameters
```

Comment:
I think that default initialization was added in F95 standard, we support only F90 (with some exceptions regarding intrinsic functions).

Reply:
Hi @laurent-altr, you might be right; it could indeed be F2003. I’ve removed the default initialization and now initialize the %EOS member within the hm_read_eos.F subroutine instead.